pkg/client/unversioned/clientcmd/api/types.go

```diff
@@ -68,6 +69,8 @@ type Cluster struct {
 	InsecureSkipTLSVerify bool `json:"insecure-skip-tls-verify,omitempty"`
 	// CertificateAuthority is the path to a cert file for the certificate authority.
 	CertificateAuthority string `json:"certificate-authority,omitempty"`
+	// VersionLabels
+	VersionLabels string `json:"version-labels,omitempty"`
```

Comment:
Yes.  It would be nice to have the issue fixed by default instead of the user needing to override the defaults to get the intended behavior.